[2025-03-29T14:57:22.024+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-29T14:57:22.224+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fist_dag_using_python_operator_v06.get_name scheduled__2025-02-03T00:00:00+00:00 [queued]>
[2025-03-29T14:57:22.294+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fist_dag_using_python_operator_v06.get_name scheduled__2025-02-03T00:00:00+00:00 [queued]>
[2025-03-29T14:57:22.297+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-03-29T14:57:22.444+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_name> on 2025-02-03 00:00:00+00:00
[2025-03-29T14:57:22.470+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fist_dag_using_python_operator_v06', 'get_name', 'scheduled__2025-02-03T00:00:00+00:00', '--job-id', '410', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpoyhaqyot']
[2025-03-29T14:57:22.523+0000] {standard_task_runner.py:105} INFO - Job 410: Subtask get_name
[2025-03-29T14:57:22.477+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1179) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-29T14:57:22.535+0000] {standard_task_runner.py:72} INFO - Started process 1204 to run task
[2025-03-29T14:57:22.745+0000] {task_command.py:467} INFO - Running <TaskInstance: fist_dag_using_python_operator_v06.get_name scheduled__2025-02-03T00:00:00+00:00 [running]> on host ba06481f48a9
[2025-03-29T14:57:23.224+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ARest' AIRFLOW_CTX_DAG_ID='fist_dag_using_python_operator_v06' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-02-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-03T00:00:00+00:00'
[2025-03-29T14:57:23.229+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-29T14:57:23.480+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-29T14:57:23.526+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-29T14:57:23.528+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fist_dag_using_python_operator_v06, task_id=get_name, run_id=scheduled__2025-02-03T00:00:00+00:00, execution_date=20250203T000000, start_date=20250329T145722, end_date=20250329T145723
[2025-03-29T14:57:23.687+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-29T14:57:23.773+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-29T14:57:23.814+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
